developer/src/kmc-generate/src/generator-messages.ts

```diff
@@ -61,6 +61,7 @@ export class GeneratorMessages {
     `The specified target platform '${def(o.target)}' is not a valid target.`
   );
 
+  // See also copier-messages.ERROR_InvalidKeyboardId
```

Comment:
What about for ERROR_InvalidLdmlKeyboardId?

Reply:
I don't understand the question, sorry!

Reply:
Just observing that `ERROR_InvalidLdmlKeyboardId` doesn't have a corresponding comment like
```
// See also copier-messages.ERROR_InvalidLdmlKeyboardId
```

Reply:
OK, because there is no corresponding message in copier :grin: